content/docs/guide/content.md

```diff
@@ -218,6 +218,25 @@ while storing your data.
 $ knoxite store -e aes latest data/ -d "AES encrypted snapshot"
 ```
 
+### Change password
+Since the repository version 4, that came with [PR 116](https://github.com/knoxite/knoxite/pull/116), you can change the password of a repository.
+To make this possible, we've added an encryption key within the encrypted repository file, which your data will be encrypted with.
+Knoxite encrypts the repository.knoxite file with the password you set when initializing the repository.
```

Comment:
Nitpick: Maybe we should denote the `repository.knoxite` file as code in markdown to emphasize it.